test/check-progress

```diff
@@ -40,6 +40,16 @@ class TestInstallationProgress(anacondalib.VirtInstallMachineCase):
         i.reach(i.steps.REVIEW)
         i.begin_installation()
 
+        with b.wait_timeout(300):
```

Comment:
This yells flaky to me :D Is there some magic that makes sure that these messages will be seen on time and reliably?

Reply:
It's really waiting for the installation messages to appear. The installation can easily take 5mins to finish, so show all messages.